docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
```

Comment:
No, no restrictions luckily. It just has progressed through versions quickly. I will upgrade it.